src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/EmployeeController.php

```diff
@@ -478,6 +480,13 @@ protected function getEmployeeFormHandler()
     protected function getErrorMessages(Exception $e)
     {
         return [
+            UploadedImageConstraintException::class => $this->trans(
+                'Image format not recognized, allowed formats are: %s',
```

Comment:
```suggestion
                'Image format not recognized, allowed format(s) is(are): .%s',
```

Reply:
Actually I just checked, this is the wording that is in the catalog, the one suggested before doesn't exist

Reply:
![image](https://user-images.githubusercontent.com/70583503/190439596-bec75ee8-ee63-4be1-adc0-5d8500d1a1f5.png)

Reply:
@jolelievre Are you sure?

Reply:
You're right @Julievrz I found `Image format not recognized, allowed formats are: %s` now I guess my catalog were not up-to-date when I searched it, sorry!